src/AssetManager/Resolver/ResolverInterface.php

```diff
@@ -9,7 +9,7 @@ interface ResolverInterface
      *
      * @param   string  $path   The path to resolve.
      *
-     * @return  \Assetic\Asset\AssetInterface|null Asset instance when found, null when not.
+     * @return  \Assetic\Contracts\Asset\AssetInterface|null Asset instance when found, null when not.
```

Comment:
Important: while it is indeed a third-party dependency that changes, this calls for a major version update.

Reply:
I agree, as I mentioned before, this is a BC break. Also, some functionalities are removed because not available in the new assetic library.